config/bundles.php

```diff
@@ -13,7 +13,7 @@
     Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle::class => ['dev' => true, 'docker' => true, 'test' => true],
     Nelmio\Alice\Bridge\Symfony\NelmioAliceBundle::class => ['dev' => true, 'docker' => true, 'test' => true],
     Fidry\AliceDataFixtures\Bridge\Symfony\FidryAliceDataFixturesBundle::class => [
-        'dev' => true, 'docker' => true, 'test' => true,
+        'dev' => true, 'docker' => true, 'test' => true
```

Comment:
is this change needed?

Reply:
Not at all, it was just a merge problem, fixed!